php/Blocks/Loader.php

```diff
@@ -112,25 +99,10 @@ public function get_data( $key ) {
 		return apply_filters( "genesis_custom_blocks_data_{$key}", $data );
 	}
 
-	/**
-	 * Gets the callback for an action or filter.
-	 *
-	 * Enables keeping these methods protected,
-	 * while allowing actions and filters to call them.
-	 *
-	 * @param string $method_name The name of the method to get the callback for.
-	 * @return callable An enclosure that calls the function.
-	 */
-	protected function get_callback( $method_name ) {
```

Comment:
Not related to this PR, but it's not necessary.

I wanted to keep methods private, and use `->get_callback` to expose them in `add_action()` and `add_filter()`.

But it's verbose and unnecessary 🤣